src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1350,7 +1353,7 @@ internal TextWriter ConsoleTextWriter
 
         private uint Run(CommandLineParameterParser cpp, bool isPrestartWarned)
         {
-            Dbg.Assert(null != cpp, "CommandLine parameter parser cannot be null.");
+            Dbg.Assert(cpp != null, "CommandLine parameter parser cannot be null.");
```

Comment:
Should we replace the assert with throw?

Reply:
No strong opinion on this one, this line was only changed because the code style checker flagged it up that null was on the left hand side (which is only necessary in C/C++ but not C#)